### PR TITLE
ESLint Pager & Pill

### PR DESCRIPTION
**Pull request by SeanArmstrong**

### CODE REVIEW

File: src/components/pager/__spec__.js [hunk 1]
```diff
@@ -114,6 +114,13 @@ describe('Pager', () => {
       });
     });
 
+    describe('when element is not incorrect', () => {
+      it('does nt call onPagination', () => {
```
Comment (ianoxley) [inline src/components/pager/__spec__.js]: Typo

File: src/components/pager/__spec__.js [hunk 2]
```diff
@@ -114,6 +114,13 @@ describe('Pager', () => {
       });
     });
 
+    describe('when element is not incorrect', () => {
```
Comment (toni-leigh) [inline src/components/pager/__spec__.js]: `when element is not incorrect it does not call onPagination` contains three negatives, can it be reworded to be clearer?

**Comment by ianrobsonsage** [comment]
manual qa passed, tested in all browsers

File: src/components/pager/__spec__.js (hunk 2 above)
Comment (SeanArmstrong) [inline src/components/pager/__spec__.js]: I think this is just a typo. Good spot

**Pull request merged**